web/pages/Chat/ChatDetail.tsx

```diff
@@ -263,22 +249,20 @@ const SwipeMessage: Component<{
   pos: number
 }> = (props) => {
   return (
-    <div class="flex h-6 w-full items-center justify-between text-[var(--text-800)]">
-      <Show when={props.list.length > 1}>
-        <div class="cursor:pointer hover:text-[var(--text-900)]">
+    <div class="flex h-6 my-1 w-full items-center justify-between text-white/50">
+        <div class={`${props.list.length > 1 ? "visible" : "invisible"} cursor:pointer hover:text-white`}>
           <Button schema="clear" onClick={props.prev}>
             <ChevronLeft />
           </Button>
         </div>
-        <div class="text-[var(--text-800)]">
-          {props.pos + 1} / {props.list.length}
+        <div class={`${props.list.length > 1 ? "visible" : "invisible"} text-white/40`}>
```

Comment:
I've added a light theme.
`text-900` is the most visible to `text-100` being the least visible.
So here you'd want something like `text-[var(--text-500)]`